components/social_links.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import twitterIcon from '../images/icons/twitter.svg';
+import githubIcon from '../images/icons/github.svg';
+import dribbbleIcon from '../images/icons/dribbble.svg';
+
+const Icons = {
+  twitter: twitterIcon,
+  github: githubIcon,
+  dribbble: dribbbleIcon,
+};
+
+export default class SocialLinks extends React.Component {
+  static propTypes = {
+    twitter: PropTypes.string,
+    github: PropTypes.string,
+    dribbble: PropTypes.string,
+  }
+
+  static defaultProps = {
+    twitter: null,
+    github: null,
+    dribbble: null,
```

Comment:
if props are specified in `propTypes` but are not required, they need to be in `defaultProps` as well, so even if it was the default, this would still be here